Assignment/solution_student/css/style.css

```diff
@@ -25,25 +25,283 @@ html {
     display: table;
 }
 .clearfix:after {
-  clear: both;
+		clear: both;
 }
 /******************************************
 /* BASE STYLES
 /*******************************************/
+
 body {
-  color: black;
-  font-size: 16px;
-  background: white;
+  	color: #033048;
+  	font-size: 16px;
+  	background-color: #F2F6F5;
+  	font-family: 'Open Sans', Tahoma, Arial;
 }
 
-/******************************************
-/* LAYOUT
-/*******************************************/
+article img {
+	width: 100%;
+	padding: 12px 0;
+}
 
-/******************************************
-/* PAGES
-/*******************************************/
+h2 {
+	font-size: 32px;
+	font-weight: 800;
+	margin-bottom: 10px;
+}
+
+h2 li {
+	font-size: 24px;
+	font-weight: 300;
+	list-style-type: none;
+}
+
+p {
+	margin: 10px 0;
+	font-size: 24px;
+	font-weight: 300;
+}
+
+aside p {
+	font-size: 22px;
+	font-weight: 300;
+}
+
+a p {
+	text-align: right;
+}
+
+a {
+	text-decoration: none;
+}
 
 /******************************************
 /* SECTIONS
-/*******************************************/
\ No newline at end of file
+/******************************************/
+
+/*MAIN CONTENT*/
+.main-content {
+	margin: 0 auto;
+	max-width: 1000px;
+}
+
+/*HEADER SECTION*/
+
+.header-content {
+	margin: 0 auto;
+	max-width: 1000px;
+}
+
+.logo {
+	float: left;	
+	color: #fff;
+	font-size: 2em;
+	font-weight: 800;
+	padding: 40px 0 40px 40px;
+	transition: color 0.5s ease;
+
+}
+
+.logo:hover {
+	color: #FDF5B0;
+	cursor: pointer;
+}
+
+header {
+	width: 100%;
+	background-color: #033048;
+}
+
+nav {
+	float: right;
+	margin-top: 30px;
+	padding-right: 40px;
+	display: inline-block;
+}
+
+nav ul {
+	color: #f9e42e;
+	font-weight: 600;
+	list-style-type: none;
+}
+
+nav li {
+	padding: 0 10px;
+	font-size: 24px;
+	display: inline-block;
+}
+
+nav a {
+	color: #f9e42e;
+	text-decoration: none;
+}
+
+nav a:hover {
+	color: #FDF5B0;
+}
+
+
+/*ARTICLE SECTION*/
+
+.article-container {
```

Comment:
Your CSS is like art! It looks sooooo good, and perfectly organized! The only thing I'd really change is how you floated the columns. You floated the `.article-container` here, which is a grandchild of the `main-content` container that neighbors the `aside`, which is the other column — so you're floating something inside a non-floated column, which works okay as you see, but what would be more clear to me, and what would make for a better setup to be completely fluid (% based rather than max-width in px), is float the two adjacent conainers, which in this case are the `.main-content` element and the `aside` element. This way, you can float them each, and give them each a % width, which would be a 5 of their container (which you should add to clearfix them). You'd also have to remove the left and right margin on them to maintain their proper %-based width (or subtract it from their width). If this sounds confusing I made some revisions that you can see here: https://gist.github.com/poopsplat/b055ea54f4f1ee8be696 + https://gist.github.com/poopsplat/e8c2e310a7cb1480cbe5